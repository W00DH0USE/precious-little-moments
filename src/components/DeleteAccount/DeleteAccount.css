.delete-account-form {
  background-color: #fff;
  height: 300px;
}