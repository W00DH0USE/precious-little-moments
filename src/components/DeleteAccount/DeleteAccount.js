import React from 'react';
import { Form, Button } from "../Utils/Utils";
import './DeleteAccount.css';

function DeleteAccount(props) {
  return (
    <div className='modal-wrapper deletePost'>
      <div className="modal-content">
        <Form className='delete-account-form' onSubmit={props.handleDelete}>
          <div className="delete-account-form-title">
            <h2 className="Form__title">Precious Little Moment: <span><em>delete account</em></span></h2>
            <hr />
            <h3 className='delete-post-title'>Delete Your Account?</h3>
            <p className="delete-warning">*Please make sure you would like to delete your account as your moment's cannot be recovered!*</p>
          </div>
          <Button className="button demo-button button-cancel" type="submit">Delete</Button>
          <Button className="button demo-button button-continue" onClick={props.close}>Cancel</Button>
        </Form>
      </div>
    </div>
  )
}

export default DeleteAccount;