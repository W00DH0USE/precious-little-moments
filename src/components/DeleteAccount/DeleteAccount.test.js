import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import DeleteAccount from './DeleteAccount';


describe(`Delete Account Component`, () => {
  it('renders without crashing', () => {
    const wrapper = shallow(<DeleteAccount />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});