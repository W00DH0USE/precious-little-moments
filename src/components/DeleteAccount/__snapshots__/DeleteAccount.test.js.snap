// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Delete Account Component renders without crashing 1`] = `
<div
  className="modal-wrapper deletePost"
>
  <div
    className="modal-content"
  >
    <Form
      className="delete-account-form"
    >
      <div
        className="delete-account-form-title"
      >
        <h2
          className="Form__title"
        >
          Precious Little Moment: 
          <span>
            <em>
              delete account
            </em>
          </span>
        </h2>
        <hr />
        <h3
          className="delete-post-title"
        >
          Delete Your Account?
        </h3>
        <p
          className="delete-warning"
        >
          *Please make sure you would like to delete your account as your moment's cannot be recovered!*
        </p>
      </div>
      <Button
        className="button demo-button button-cancel"
        type="submit"
      >
        Delete
      </Button>
      <Button
        className="button demo-button button-continue"
      >
        Cancel
      </Button>
    </Form>
  </div>
</div>
`;
